great_expectations/validator/validation_graph.py

```diff
@@ -24,7 +24,7 @@ def __init__(
         self._metric_value_kwargs = metric_value_kwargs
         if metric_dependencies is None:
             metric_dependencies = {}
-        self.metric_dependencies = metric_dependencies
+        self._metric_dependencies = metric_dependencies
```

Comment:
Since we are not doing anything special in the setter and getter can we keep this attribute public?

Reply:
@anthonyburdi We can -- if you think we should.  Personally, I like the consistent property setter/getter as a style and keep fields private.  Thoughts?

Reply:
Totally a stylistic call, totally up to you and this won't gate my approval. My rationale is that keeping the attribute public with default setters and getters keeps the class less verbose, with the main benefit that setters and getters with custom logic are more obvious.